UnitTests/Directory.Build.props

```diff
@@ -1,5 +1,6 @@
 <Project>
   <Import Project="..\Directory.Build.props" />
```

Comment:
💡 This works, but the most "copy-paste-friendly" way to add this line in non-root files is this:

```
<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)\.., Directory.Build.props))\Directory.Build.props" />
```